bin/reportdaily.py

```diff
@@ -1,15 +1,24 @@
 #!/usr/bin/env python3
 
+
 import argparse
 import logging
 from logging.config import dictConfig
+from pickle import TRUE
```

Comment:
Please don't do that. It's confusing as you have `True` (builtin) and `TRUE` now. In this case I would really recommend to use the longer, but more obvious form `import pickle` and then `pickle.TRUE`.

After looking through your source code, it seems you don't use it. So better remove it:

```suggestion
```

Reply:
Hmm strange, i need to check my vs code settings,  it seems like sometimes vs codes auto adds imports if something is missing.
I need to disable this feature. 
I cant remeber to import "True" 
Thanks for the tipp.